examples/proxy/record-encryption/load-balancer/proxy/proxy-config.yaml

```diff
@@ -19,7 +19,7 @@ data:
           config:
             # this is the address the clients will refer to in their bootstrap.servers configuration.
             bootstrapAddress: localhost:9092
-            brokerAddressPattern: a5babde172a1c417dbd4fd7ecc64625f-c70f3d074bf71d17.elb.us-east-1.amazonaws.com
+            brokerAddressPattern: af0898e4283d84513ae3fbb14bd642c8-9707b526d142d05b.elb.us-east-1.amazonaws.com
```

Comment:
Is the load balancer forwarding a port for each broker so we don't need a `$(nodeId)` in the pattern?

Reply:
Though maybe using an ELB isn't a great example for user...

Reply:
Yes, that seems dumb.  Let me check that.

Reply:
no, sorry it is okay.  It doesn't need `$(nodeId)` as the routing is being done by port number.  It does work with brokers > 1 already.